bundles/binding/org.openhab.binding.zwave/database/philio/pan08.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>PAN08</Model>
+	<Label lang="en">Relay Insert for Blind Control</Label>
+	<CommandClasses>
+		<Class><id>0x20</id></Class>
+		<Class><id>0x25</id></Class>
+		<Class><id>0x27</id></Class>
+		<Class><id>0x60</id></Class>
+		<Class><id>0x70</id></Class>
+		<Class><id>0x71</id></Class>
+		<Class><id>0x72</id></Class>
+		<Class><id>0x85</id></Class>
+		<Class><id>0x86</id></Class>
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>int</Type>
+			<Minimum>1</Minimum>
+			<Maximum>32767</Maximum>
+			<WriteOnly>false</WriteOnly>
+			<Default>720</Default>
+			<Size>2</Size>
+			<Label lang="en">Watt meter report interval (5s)</Label>
+			<Help lang="en"><![CDATA[Interval (x5 seconds) to send out watt meter report]]></Help>
+		</Parameter>
+		<Parameter>
+			<Index>2</Index>
+			<Type>int</Type>
+			<Minimum>1</Minimum>
+			<Maximum>32767</Maximum>
+			<WriteOnly>false</WriteOnly>
+			<Default>6</Default>
+			<Size>2</Size>
+			<Label lang="en">KWH Meter Report Interval (10min)</Label>
+			<Help lang="en"><![CDATA[Interval (x10 minutes) to send out KWH Meter Report]]></Help>
+		</Parameter>
+		<Parameter>
+			<Index>3</Index>
+			<Type>int</Type>
+			<Minimum>1</Minimum>
+			<Maximum>1100</Maximum>
+			<WriteOnly>false</WriteOnly>
+			<Default>1100</Default>
+			<Size>2</Size>
+			<Label lang="en">Threshold for Load Caution (10mA)</Label>
+			<Help lang="en"><![CDATA[Threshold (x0.01A) of current for Load Caution]]></Help>
+		</Parameter>
+		<Parameter>
+			<Index>4</Index>
+			<Type>int</Type>
+			<Minimum>1</Minimum>
+			<Maximum>10000</Maximum>
+			<WriteOnly>false</WriteOnly>
+			<Default>10000</Default>
+			<Size>2</Size>
+			<Label lang="en">Threshold of KWh for Load Caution</Label>
+		</Parameter>
+	</Configuration>
+	<Associations>
+		<Group>
```

Comment:
Actually, I'm not sure what this group is for. I left it in as it is listed in the manual and the related PAN06 has them.